magnet/excitation-data/si-corrector-wig180.txt

```diff
@@ -0,0 +1,68 @@
+# HEADER
+# ======
+# label             si-corrector-wig180
+# harmonics         0
+# main_harmonic     0 normal
+# rescaling_factor  1.0
+# units             Ampere  T*m T*m
+
+# EXCITATION DATA
+# ===============
+-005.000  -1.8111e-03 +0.0000e+00
++005.000  +1.8522e-03 +0.0000e+00
+
+# COMMENTS
+# ========
+# 1. tables needs to be updated with measurement!
```

Comment:
need update?

Reply:
an update (also deleting this comment) is coming up...